src/main/scala/com/campudus/tableaux/database/model/structure/ColumnModel.scala

```diff
@@ -694,30 +769,71 @@ class ColumnModel(val connection: DatabaseConnection) extends DatabaseQuery {
       (t, _) <- t.query(s"ALTER TABLE user_table_lang_$tableId DROP COLUMN IF EXISTS column_$columnId")
 
-      (t, _) <- t.query("DELETE FROM system_columns WHERE column_id = ? AND table_id = ?", Json.arr(columnId, tableId)).map({ case (t, json) => (t, deleteNotNull(json)) })
+      (t, _) <- t
+        .query("DELETE FROM system_columns WHERE column_id = ? AND table_id = ?", Json.arr(columnId, tableId))
+        .map({ case (t, json) => (t, deleteNotNull(json)) })
 
       _ <- t.commit()
     } yield ()
   }
 
-  def change(table: Table, columnId: ColumnId, columnName: Option[String], ordering: Option[Ordering], kind: Option[TableauxDbType], identifier: Option[Boolean], displayInfos: Option[Seq[DisplayInfo]], countryCodes: Option[Seq[String]]): Future[ColumnType[_]] = {
+  def change(table: Table,
+             columnId: ColumnId,
+             columnName: Option[String],
+             ordering: Option[Ordering],
+             kind: Option[TableauxDbType],
+             identifier: Option[Boolean],
+             displayInfos: Option[Seq[DisplayInfo]],
+             countryCodes: Option[Seq[String]]): Future[ColumnType[_]] = {
     val tableId = table.id
 
     for {
       t <- connection.begin()
 
       // change column settings
-      (t, resultName) <- optionToValidFuture(columnName, t, { name: String => t.query(s"UPDATE system_columns SET user_column_name = ? WHERE table_id = ? AND column_id = ?", Json.arr(name, tableId, columnId)) })
-      (t, resultOrdering) <- optionToValidFuture(ordering, t, { ord: Ordering => t.query(s"UPDATE system_columns SET ordering = ? WHERE table_id = ? AND column_id = ?", Json.arr(ord, tableId, columnId)) })
-      (t, resultKind) <- optionToValidFuture(kind, t, { k: TableauxDbType => t.query(s"UPDATE system_columns SET column_type = ? WHERE table_id = ? AND column_id = ?", Json.arr(k.name, tableId, columnId)) })
-      (t, resultIdentifier) <- optionToValidFuture(identifier, t, { ident: Boolean => t.query(s"UPDATE system_columns SET identifier = ? WHERE table_id = ? AND column_id = ?", Json.arr(ident, tableId, columnId)) })
-      (t, resultCountryCodes) <- optionToValidFuture(countryCodes, t, { codes: Seq[String] => t.query(s"UPDATE system_columns SET country_codes = ? WHERE table_id = ? AND column_id = ?", Json.arr(Json.arr(codes: _*), tableId, columnId)) })
+      (t, resultName) <- optionToValidFuture(columnName,
+                                             t, { name: String =>
+                                               t.query(
+                                                 s"UPDATE system_columns SET user_column_name = ? WHERE table_id = ? AND column_id = ?",
+                                                 Json.arr(name, tableId, columnId)
+                                               )
+                                             })
+      (t, resultOrdering) <- optionToValidFuture(ordering, t, { ord: Ordering =>
```

Comment:
Looks okay here, but above and below is not nice. It's hard to see the pattern...